clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1271,4 +1274,35 @@ private static byte checkRange(final byte i) {
         }
         return map;
     }
+
+
+    /**
+     * Convert a ISO8601 based TimeStamp to Epoch Value
```

Comment:
Few nitpicks here:
Convert an ISO8601 based timestamp to an epoch value
....
@return epoch value of a given timestamp
@throws ParseException for timestamp that doesn't follow ISO8601 format

Reply:
This was not addressed, you disagree with the changes suggested?